tests/test_upstream_versions.py

```diff
@@ -1354,7 +1354,9 @@ def test_update_upstream_versions_run_parallel_custom_sources(
     ) == ("source a", "source b")
 
 
-@mock.patch("conda_forge_tick.update_upstream_versions.RANDOM_FRAC_TO_UPDATE", new=1.1)
+@mock.patch(
+    "conda_forge_tick.update_upstream_versions.FRAC_UPDATE_UPSTREAM_VERSIONS", new=1.1
+)
```

Comment:
Shouldn't these be patched in `settings`? I always struggle to remember how what the best practice is. It might be that the current change is the correct one.